app/models/enterprise.rb

```diff
@@ -462,6 +455,17 @@ def devise_mailer
 
   private
 
+  def validate_white_label_logo_link
+    return if white_label_logo.blank?
+
+    return if white_label_logo_link.blank?
+
+    uri = URI(white_label_logo_link)
```

Comment:
Nice. We could call `strip` on the input in case someone copies whitespace but it's not necessary.